test/JDBC/upgrade/latest/schedule

```diff
@@ -105,6 +105,7 @@ BABEL-2955
 BABEL-3358
 BABEL-3747
 BABEL-3781
+BABEL-3802
```

Comment:
this file only add major version upgrade tests, for minor upgrade testing add it to 14.6 -> 14.7 schedule file